src/DensityMatrixSparse.h

```diff
@@ -43,16 +43,14 @@ class DensityMatrixSparse
 
     ~DensityMatrixSparse();
 
-    void setUniform(const double nel, const int new_orbitals_index);
+    void setUniform(const double nel);
 
-    void setto2InvS(
-        const VariableSizeMatrix<sparserow>& invS, const int orbitals_index);
-    int getOrbitalsIndex() const { return orbitals_index_; }
-    void setMatrix(
-        const VariableSizeMatrix<sparserow>& mat, const int orbitals_index)
+    void setto2InvS(const VariableSizeMatrix<sparserow>& invS);
+    int getIndex() const { return orbitals_index_; }
+    void setMatrix(const VariableSizeMatrix<sparserow>& mat)
     {
-        *dm_            = mat;
-        orbitals_index_ = orbitals_index;
+        *dm_ = mat;
+        orbitals_index_++;
```

Comment:
@jeanlucf22 any reason why we keep this name instead of adopting the change to update_index_ ?

Reply:
fixed